.github/workflows/test-helm.yaml

```
@@ -0,0 +1,74 @@
+name: Unit and Integration Test - Helm
+
+on:
+  - push
+  - pull_request
+
+jobs:
+  test-helm-chart:
```

Can we keep it consistent with Job names.
```suggestion
  test:
```

---

Done.